linux_os/guide/system/software/updating/timer_dnf-automatic_enabled/rule.yml

```diff
@@ -23,3 +23,9 @@ references:
 ocil_clause: 'the dnf-automatic.timer is not enabled'
 
 ocil: "{{{ ocil_timer_enabled(timer='dnf-automatic') }}}"
+
+template:
+    name: timer_enabled
+    vars:
+        package_name: dnf-automatic
```

Comment:
@yuumasato It's PACKAGENAME in the templates for timer_enabled, so the key should ideally be without the `_`.

Reply:
Fixed, diff data can be seen in 0ae0e65cbaa61fcf23eac67283a8fbd96e24a6ab